# message type for pendulum angle

float64 position
float64 velocity

